Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/StateSynchronizationSceneManager.cs

```diff
@@ -0,0 +1,480 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.Socketer;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    internal class StateSynchronizationSceneManager : Singleton<StateSynchronizationSceneManager>
+    {
+        private const int FrameVotesUntilChangingFrameSkipCount = 10;
+        private const int MaximumQueuedByteCount = 2048;
+        private const float QueueAgeMillisecondsPerFrameSkip = 0.05f;
+        private int framesToSkipForBuffering = 0;
+        private int frameSkippingStride = 1;
+        private int frameSkippingVoteCounter = 0;
+        private short nextLocallyUniqueId;
+        private bool shouldClearScene;
+
+        public const string DefaultStateSynchronizationPerformanceParametersPrefabName = "DefaultStateSynchronizationPerformanceParameters";
+        public const string CustomNetworkServicesPrefabName = "CustomNetworkServices";
```

Comment:
>CustomNetworkServicesPrefabName [](start = 28, length = 31)

In the new naming scheme, maybe this could be called "CustomBroadcasterServices"